pkg/menu/coomplete_menu.go

```diff
@@ -57,6 +59,20 @@ func GenerateCompleteMenu(kc client.Client, disco discovery.ServerResourcesInter
 				Name: menuoutlines.MenuSectionName(group),
 			},
 		}
+
+		// Kubedb -> KubeDB
+		if strings.Contains(strings.ToLower(sec.Name), "kubedb") {
+			sec.Name = strings.Replace(sec.Name, "Kubedb", "KubeDB", -1)
```

Comment:
Possible to get `kubedb` somehow ? 
In that case, `sec.Name = strings.Replace(sec.Name, "kubedb", "KubeDB", -1)`

Reply:
Here, I am converting the entire string to lowercase to check if it contains the word 'kubedb'. 'Kubedb Op' contains the word 'kubedb', it will be replaced by 'KubeDB'.

Reply:
It is clear from the code.  
My question was , Is it possible to get lowercase `kubedb` in sec.Name, before the replace() ? 
If yes, we should add another line after 65, `sec.Name = strings.Replace(sec.Name, "kubedb", "KubeDB", -1)`.

Because, `kubedb <random-string>` will remain same, if we don't do so.